src/main/scala/com/gu/newsletterlistcleanse/Lambda.scala

```diff
@@ -37,22 +41,31 @@ class Lambda {
   val config: NewsletterConfig = NewsletterConfig.load(credentialProvider)
   val s3Client: AmazonS3 = AmazonS3ClientBuilder.standard
     .withCredentials(credentialProvider)
-    .withRegion(Regions.EU_WEST_1).build
+    .withRegion(Regions.EU_WEST_1)
+    .build()
+  val snsClient: AmazonSNS = AmazonSNSClientBuilder.standard
+    .withCredentials(credentialProvider)
+    .withRegion(Regions.EU_WEST_1)
+    .build()
   val databaseOperations: DatabaseOperations = new BigQueryOperations(config.serviceAccount, config.projectId)
   val newsletters: NewslettersApiClient = new NewslettersApiClient()
 
   val cutOffDatesService = new CutOffDatesService(databaseOperations)
   val cleanseListService = new CleanseListService(config, s3Client, databaseOperations)
   val brazeService = new BrazeUsersService(config)
+  val reportService = new ReportService(snsClient, config)
 
   def handler(lambdaInput: GetCutOffDatesLambdaInput, context: Context): Unit = {
     val env = Env()
+    val dryRun = config.dryRun || lambdaInput.dryRun
```

Comment:
I was thinking about completely removing the dryRun from the config to make it less confusing, but wasn't sure

Reply:
I think it's good to have it there as a manual override in case we ever wanted to turn off the actual cleanse but keep generating reports.